extension/inet/inet_escape_functions.cpp

```diff
@@ -0,0 +1,491 @@
+#include "duckdb/planner/expression/bound_function_expression.hpp"
+#include "duckdb/common/types/blob.hpp"
+#include "duckdb/main/extension_util.hpp"
+#include "inet_extension.hpp"
+#include "inet_functions.hpp"
+#include "html_charref.hpp"
+
+namespace duckdb {
+
+struct UnescapeBindData : public FunctionData {
+	const HTML5NameCharrefs html5_names;
+	const unordered_set<uint32_t> invalid_codepoints;
+	const unordered_map<uint32_t, const char *> special_invalid_charrefs;
+
+	UnescapeBindData()
+	    : html5_names(HTML5NameCharrefs()), invalid_codepoints(ReturnInvalidCodepoints()),
+	      special_invalid_charrefs(ReturnInvalidCharrefs()) {
+	}
+
+	unique_ptr<FunctionData> Copy() const override {
+		return make_uniq<UnescapeBindData>();
+	}
+
+	bool Equals(const FunctionData &other_p) const override {
+		auto &other = other_p.Cast<UnescapeBindData>();
```

Comment:
Since we only have one constructor, and everything is const, I feel like this can be wrapped in an `#ifdef DEBUG` .. `#endif`
And we just return `true`, because it can realistically never be false.

The only thing that could fail is the `Cast`, which we probably want to turn into a `dynamic_cast` call so we can return false if it fails, instead of throwing an InternalException which this will do

Reply:
To be honest I don't really know what Equals is used for in practice, but I suspect this is of little importance 👍